index.html

```diff
@@ -45,6 +45,155 @@ <h3 class="contact">
         </ul>
         
     </section>
+
+    <section class="mid-container">
+        <div class="work-card">
+            <img class="card-image" src="images/work-card1.svg" alt="project-1">
+            <h1 class="card-head">Tonic</h1>
```

Comment:
It is not considered good practice to have multiple `h1` tags across a web page. Usually, just one `h1` tag it's enough for a web page.

- [ ] Kindly, replace each `h1` tag from this "mid-container" section in order to keep up with the good practices.

Reply:
Can I use h2 - h6 instead or should use the p tag?